packages/coordinator/src/components/Popup/Popup.tsx

```diff
@@ -43,6 +44,9 @@ export function Popup(props: PopupProps) {
       textAlign: "center",
       paddingBottom: "10px",
     },
+    bottom: {
+      paddingBottom: "30px",
```

Comment:
Do we want this in all popups? Or just the delete appointment one?

Reply:
all pf them

Reply:
Did you check how the other popups look after this change? Do they look OK?

Reply:
Good point :) yes checked them 
BTW we wanted this behavior when we originally wrote the pop-up but there was an issue so it was reverted

Reply:
Great.

Yedidya just pushed another usage of the popup, please take a look at it as well, and if it looks good let's merge this :)